src/delagent/ui/admin-upload-delete.php

```diff
@@ -190,6 +193,14 @@ public function Output()
     $text = _("Delete");
     $V.= "<input type='submit' value='$text'>\n";
     $V.= "</form>\n";
+
+    $V.= <<<EOT
```

Comment:
why do you use heredoc syntax here, and [at other places](https://github.com/siemens/fossology/blob/0159a0f38590e5d8739e401619da039b9b9d6102/src/www/ui/admin-dashboard.php#L129-L165) the usual multiline string syntax?